lmn/templates/lmn/shows/show_detail.html

```diff
@@ -1,8 +1,22 @@
 {% extends 'lmn/base.html' %}
 {% block content %}
 
-<h2>Show Detail</h2>
+<h2>{{ show.artist.name }} @ {{ show.venue.name }} {{ show.show_date }}</h2>
```

Comment:
Potentially having venue name _on_ show date would be good, but that's just nitpicking.